packages/core/src/components/ChoiceList/ChoiceList.test.jsx

```diff
@@ -166,18 +166,37 @@ describe('ChoiceList', () => {
         .first()
         .simulate('change');
 
-      expect(onChange.mock.calls.length).toBe(1);
+      expect(onChange).toHaveBeenCalled();
     });
 
     it('calls onBlur', () => {
       const onBlur = jest.fn();
-      const data = shallowRender({ onBlur });
+      const onComponentBlur = jest.fn();
+      const data = shallowRender({ onBlur, onComponentBlur });
       data.wrapper
         .find('Choice')
         .first()
         .simulate('blur');
 
-      expect(onBlur.mock.calls.length).toBe(1);
+      expect(onBlur).toHaveBeenCalled();
+      setTimeout(() => {
+        expect(onComponentBlur).not.toHaveBeenCalled();
```

Comment:
I don't actually understand why this one is working and feel like it must be that when we simulate blur in Enzyme, it focuses on the next one automatically. It works when I test it out for real though.

Reply:
When we blur the first choice, it does go to the next choice automatically, but we dont want it to fire `onComponentBlur` until the last choice is blurred. Does that answer your question?

Reply:
My question is whether this is an implicit feature of Enzyme. If we're all okay with this as expected behavior of Enzyme, then that's good enough for me. Might be good to add a comment that says that if we simulate blur that it will automatically focus on the next element.

Reply:
Ah I see, I couldnt find much info on the enzyme docs page to confirm, i just assumed it was expected cuz thats what the browser does. I can add a comment though for sure